tests/unit/sagemaker/serve/mode/test_in_process_mode.py

```diff
@@ -0,0 +1,166 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
```

Comment:
Is there an overlap between 1/2 and this CR?

Reply:
Yes because 1/2 is the broader in_process_mode and it has the functions stubbed, 2/2 has in_process + fastapi